example/code/glob_avg.py

```diff
@@ -0,0 +1,55 @@
+'''
+Creates a monthly temparature time series for a global average
+temperature, using of all Berkeley region data per time index. The
+average is of the temperature absolute value, the anomaly plus the
```

Comment:
Not the absolute value. "The script combines the temperature anomaly with the climatology for each time/region."?

Reply:
Removed the comment